<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>League Window Fix</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>league.ico</normaloff>league.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>League Window Fix by SpaceManiac - http://wombat.platymuus.com/</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="versionLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Version 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QTextBrowser" name="logText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="startAutofix">
        <property name="text">
         <string>Start Autofix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopAutofix">
        <property name="text">
         <string>Stop Autofix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detectValues">
        <property name="text">
         <string>Detect Values</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="position">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="windowX"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="windowY"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="windowW"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="windowH"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="wLabel">
          <property name="text">
           <string>W</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="hLabel">
          <property name="text">
           <string>H</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="moveWindow">
        <property name="text">
         <string>Move Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minimizeToTray">
        <property name="text">
         <string>Minimize to Tray</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startAutofix</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startAutofix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopAutofix</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopAutofix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detectValues</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>detectValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveWindow</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moveWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimizeToTray</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeToTray()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startAutofix()</slot>
  <slot>stopAutofix()</slot>
  <slot>detectValues()</slot>
  <slot>moveWindow()</slot>
  <slot>minimizeToTray()</slot>
 </slots>
</ui>
